docs/topics/auth/default.txt

```diff
@@ -999,6 +999,9 @@ implementation details see :ref:`using-the-views`.
       authenticated users accessing the login page will be redirected as if
       they had just successfully logged in. Defaults to ``False``.
 
+    * ``success_url_allowed_hosts``: A set of additional hosts that are safe
```

Comment:
Missing versionchanged annotations and release notes. 
It might not be clear what "additional" is in context to.